apps/etterlatte-oppdater-behandling/src/main/kotlin/BehandlingService.kt

```diff
@@ -43,6 +43,7 @@ import no.nav.etterlatte.libs.common.sak.SakId
 import no.nav.etterlatte.libs.common.sak.Saker
 import no.nav.etterlatte.libs.common.tidspunkt.Tidspunkt
 import no.nav.etterlatte.libs.ktor.route.FoedselsnummerDTO
+import no.nav.etterlatte.libs.ktor.route.logger
```

Comment:
Den logger-en som kjem importert her trur eg ikkje vi burde eksportert i det heile tatt, for den har namnet "TilgangsSjekk", men det er ein uvane vi har lagt oss til ein del plassar, med å deklarere loggers som public der vi eigentleg vil ha dei innkapsla.

Ei betre løysing er å eksplisitt definere opp logger i klassa her - `private val logger = LoggerFactory.getLogger(this::class.java)`